src/components/forms/CloudInitForm.tsx

```diff
@@ -29,12 +32,76 @@ interface Props {
 }
 
 const CloudInitForm: FC<Props> = ({ formik }) => {
-  const codeRenderer = (value?: unknown) =>
-    value === "-" || value === undefined ? (
-      ""
-    ) : (
-      <CloudInitConfig config={value as string} />
-    );
+  const getCloudInitRow = (label: string, name: string, value?: string) => {
+    const metadata = getConfigRowMetadata(formik.values, name);
+    const isOverridden = value !== undefined;
+
+    return getConfigurationRowBase({
+      configuration: <b>{label}</b>,
+      inherited: (
+        <div
+          className={classnames({
+            "u-text--muted": isOverridden,
+            "u-text--line-through": isOverridden,
+          })}
+        >
+          <div className="mono-font">
+            <b>
```

Comment:
what's the `<b>` tag for?

Reply:
We should use strong instead. b is to make things bold, but more legacy.